### ISSUE TITLE
incorrect copyright text when ipr="noModificationTrust200902"

### ISSUE THREAD

**Issue by sginoza** [ietf-tools/xml2rfc#949]
I don't think the BSD license text is supposed to be included in the copyright material when ipr="noModificationTrust200902", per Section 4.c of the TLP <https://trustee.ietf.org/documents/trust-legal-provisions/tlp-5/>: 

> c. License. In addition to the licenses granted under Section 3, unless one of the legends contained in Section 6.c.i or 6.c.ii is included in an IETF Document containing Code Components, such Code Components are also licensed to each person who wishes to receive such a license on the terms of the “Revised BSD License”, as described below.

**Comment by jrlevine** [comment]
You're right. The BSD code license only applies to documents with the normal TLP license, not to the no-modification one.

This bit us earlier this year when the robots.txt draft had the no-modification license and tried to include a code section.

**Comment by kesara** [comment]
@sginoza Do you have an example?

If `submissionType` is `IETF`<sup>*</sup>, `xml2rfc` adds the following text regardless of the  `ipr` status:
```
            Copyright (c) {year} IETF Trust and the persons identified as the
            document authors. All rights reserved.
            
            This document is subject to BCP 78 and the IETF Trust's Legal
            Provisions Relating to IETF Documents
            (<eref target="{scheme}://trustee.ietf.org/license-info"/>) in effect on the date of
            publication of this document. Please review these documents
            carefully, as they describe your rights and restrictions with
            respect to this document. Code Components extracted from this
            document must include Revised BSD License text as described in
            Section 4.e of the Trust Legal Provisions and are provided without
            warranty as described in the Revised BSD License.
```
Is this behaviour correct?

To avoid the code components license notice, you can set the `submissionType="independent"`.

<sup>*</sup> If `submissionType` is not set default is `IETF`.
